src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -456,6 +457,23 @@ class BuildArgument(cli.Argument):
             'required': False,
         },
     )
+    BETA_GROUP_NAMES_REQUIRED = cli.ArgumentProperties(
+        key='beta_group_names',
+        flags=('--beta-group-names',),
```

Comment:
I think `--beta-group` would be better suited here. Flags with `nargs` support are still defined in singular form and the _name_ part in flag doesn't give much value, but makes it more inconvenient to write. Docs/help message already says that this takes the name of the group.